client/main.lua

```diff
@@ -214,4 +202,21 @@ RegisterNUICallback('removeCharacter', function(data, cb)
     DeletePed(charPed)
     TriggerEvent('qb-multicharacter:client:chooseChar')
     cb("ok")
-end)
\ No newline at end of file
+end)
+
+--- Low level game event triggered when the player started the session.
+---@param name string - Name of the event https://docs.fivem.net/docs/scripting-reference/events/list/gameEventTriggered/
+AddEventHandler("gameEventTriggered", function(name)
+    if name == "CEventNetworkStartSession" then
```

Comment:
```suggestion
---This event is fired after low-level game events take place.
---https://docs.fivem.net/docs/scripting-reference/events/list/gameEventTriggered/
---@param name string  the game event name that was triggered
AddEventHandler("gameEventTriggered", function(name)
    if name == "CEventNetworkStartSession" then -- when a network session has started
```
The comment above `gameEventTriggered` should be relevant to itself, not `CEventNetworkStartSession`.

Reply:
in understand but that comment is just a copy and paste of the web page, need to be something else because doesn't explain much.

Reply:
This comment is indeed a copy paste of the web page, but it does not need to be anything else, it explains exactly what it needs